utils/sysfs/sysfs.go

```diff
@@ -164,6 +171,24 @@ func (fs *realSysFs) GetCPUPhysicalPackageID(cpuPath string) (string, error) {
 	return strings.TrimSpace(string(packageID)), err
 }
 
+func (fs *realSysFs) GetBookID(cpuPath string) (string, error) {
+	bookIDFilePath := fmt.Sprintf("%s%s", cpuPath, bookIDFilePath)
+	bookID, err := os.ReadFile(bookIDFilePath)
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(bookID)), err
```

Comment:
nit, the err could be nil here.

Reply:
err = nil, is fine here. It will be treated as no error in the follow-up/caller code.

Reply:
Very true; however, reading the code requires an second glance if 'err' is correct within the context, instead of visually seeing a nil. It takes slightly longer to review.

Reply:
All good. 
Can we launch tests also on this one?

Reply:
line 180, let's change the err to a nil.

Reply:
done